builtin/providers/aws/provider.go

```diff
@@ -172,6 +172,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_vpc":                      dataSourceAwsVpc(),
 			"aws_vpc_endpoint_service":     dataSourceAwsVpcEndpointService(),
 			"aws_vpc_peering_connection":   dataSourceAwsVpcPeeringConnection(),
+			"aws_kms_secrets":              dataSourceAwsKmsSecrets(),
```

Comment:
I'm still of the opinion that singular makes sense here on the basis of the interpolation syntax: `${data.aws_kms_secret.whatever}`